Assignment/Conversion.c

```diff
@@ -0,0 +1,87 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// Conversion.c
+// Program for Number Conversion.
+// ------------------------------------------------------------------------------------------------
+
+#pragma warning(disable:4996)
+#include <stdio.h>
+
+void DecToBinary (int n);
+void DecToHexa (int n);
+int mBitLength;
+/// <summary>Decimal number to binary Conversion</summary>
+void DecToBinary (int n) {
+   int binary[32], i = 0, isNegative = 0; // Flag is to check if the number is negative
+   if (n == 0) {

```

Comment:
You don't need this if check, your while loop will run only once if n == 0, that is enough.